syntax = "proto3";
package job_export;





/*
PostJobImport返回
*/
message PostJobImportResponse {
    /*
    导入结果 ok failed unknown
    */
    string status = 1 ;
}

/*
PostJobImportApi返回
*/
message PostJobImportResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    job_export.PostJobImportResponse data = 4 ;
}

